tests/api/controllers/transaction_controller_test.go

```diff
@@ -0,0 +1,19 @@
+package tests
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/numary/ledger/api/controllers"
+)
+
+func TestNewTransactionController(t *testing.T) {
```

Comment:
Not sure if those type check tests would make sense once we use fx for creation and injection